setup.py

```diff
@@ -58,6 +58,10 @@ def run(self):
     py_modules=['pdbpp', '_pdbpp_path_hack.pdb'],
     package_dir={"": "src"},
     url='http://github.com/antocuni/pdb',
+    project_urls={
+        'Bug Tracker': 'http://github.com/antocuni/pdb/issues',
+        'Source Code': 'http://github.com/antocuni/pdb',
+    },
```

Comment:
Should be pdbpp/pdbpp - I can amend it myself though.

Reply:
Thanks Daniel! Didn't notice my mistake as there is a redirection.

Hope it is fine now - otherwise just amend what's necessary.